sync/src/synchronizer/get_headers_process.rs

```diff
@@ -59,6 +59,10 @@ impl<'a> GetHeadersProcess<'a> {
             let state = self.synchronizer.shared.state();
             if let Some(flag) = state.peers().get_flag(self.peer) {
                 if flag.is_outbound || flag.is_whitelist || flag.is_protect {
+                    // Remove genesis hash on locators, because `try_update_best_known_with_unknown_header_list` can use
```

Comment:
fn `try_update_best_known_with_unknown_header_list` has been deleted in this PR, please reword the comment